pootle/apps/accounts/managers.py

```diff
@@ -97,8 +101,8 @@ def get_users_with_permission(self, permission_code, project, language):
             permissionset__positive_permissions__codename=permission_code
         )
 
-        language_path = language.directory.pootle_path
-        project_path = project.directory.pootle_path
+        language_path = "/%s/" % language.code
+        project_path = "/projects/%s/" % project.code
```

Comment:
`project.pootle_path` does the same: https://github.com/translate/pootle/blob/master/pootle/apps/pootle_project/models.py#L360-L362